test/cql-pytest/run

```diff
@@ -14,6 +14,19 @@ else:
     cmd = run.run_scylla_cmd
     check_cql = run.check_cql
 
+# If the "--vnodes" option is given, drop the "tablets" experimental
+# feature (turned on in run.py) so that all tests will be run with the
+# old vnode-based replication instead of tablets. This option only has
+# temporary usefulness, and should eventually be removed.
+if '--vnodes' in sys.argv:
+    sys.argv.remove('--vnodes')
+    def run_without_tablets(pid, dir):
+        (c, e) = run_without_tablets.orig_cmd(pid, dir)
+        c.remove('--experimental-features=tablets')
```

Comment:
This will stop working when we make tablets non-experimental, but in the meanwhile it will allow us to run tests with vnodes, so it is good enough right now.

Reply:
Right, the patch that will remove the "--experimental-features=tablets" will need to clean up all its uses - here, in test/cql-pytest/run.py, in test.py and in other places. But temporarily, it's useful.